### ISSUE TITLE
How do I record sensor data on an apple watch?

### ISSUE THREAD

**Issue by fellowProgrammer** [degtiarev/MotionCollector#3]
I am trying to record sensor data using my Apple Watch. I don't need any data recorded from the phone, just the apple watch. How would I do this? I figured out how to use the iPhone app and it works really well, however I cannot seem to figure out how to record sensor data on the Apple Watch and then send it to my computer. If someone could make a comprehensive guide on how to use the app and specifically the watch app, that would be awesome!

**Comment by degtiarev** [comment]
Run the app on iPhone -> set record id -> run Apple Watch App -> force touch -> press start.
To stop: force touch Apple Watch -> press stop. 
I hope they did not change API a lot and it still works. Good luck!

**Comment by fellowProgrammer** [comment]
I ran the app on my phone (which is paired with my apple watch) and the app installed on my phone and my watch and ran smoothly. However when I unplugged the phone from my computer and reopened the app, the link between the phone and watch does not seem to exist. And the apple watch does not seem to send the data to the phone if I flip the "Record Data from iPhone" switch to off. Any idea what could be causing these issues? Also is there a way to change the record id on the apple watch?

**Comment by degtiarev** [comment]
The app on iPhone does not work in the background. The app should be always on-screen during the work. You need to set display auto-lock-never

**Comment by fellowProgrammer** [comment]
Thanks for your help. I think I finally got the app to work reliably. However, I do have a suggestion for a future improvement. It is quite hard to use the apple watch to start and stop the sensor recording. So it would be much easier if I could remotely start the recording on the apple watch from the iPhone. I find myself bending my wrist in weird ways in order to try to keep the screen on (it is a series 4 apple watch) in order to force touch and start or stop the recording. I tried to make some crude modifications myself, but since this is the first time I have had to use the apple watch as a part of the app I am making, I am not very experienced in this field and my modifications did not seem to work.
